modules/nf-core/taxonkit/name2taxid/tests/main.nf.test

```diff
@@ -0,0 +1,100 @@
+nextflow_process {
+
+    name "Test Process TAXONKIT_NAME2TAXID"
+    script "../main.nf"
+    process "TAXONKIT_NAME2TAXID"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "untar"
+    tag "taxonkit"
+    tag "taxonkit/name2taxid"
+
+    setup {
+        run("UNTAR"){
+            script "modules/nf-core/untar/main.nf"
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file("ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz", checkIfExists: true)
```

Comment:
I have a mini set of (working) taxdump files for createtaxdb:

https://github.com/nf-core/test-datasets/tree/createtaxdb/data/taxonomy

You're welcome to tar.gz them and add to test datasets if you want to speed up tests if slow?

Reply:
The tests don't seem slow, but let me see if there's a difference.